src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -18,6 +18,57 @@ export interface StripeError {
     payment_intent: PaymentIntent;
 }
 
+/**
+ * Object definition for part of the data sent to confirm the PaymentIntent.
+ */
+export interface AddressOptions {
+    city?: string;
```

Comment:
Are these all optionals?

Reply:
All of these can be optional depending on the method and the settings with which we create the stripe element.